modules/ztp-configuring-the-hub-cluster-for-backup-and-restore.adoc

```diff
@@ -0,0 +1,156 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-configuring-the-hub-cluster-for-backup-and-restore_{context}"]
+= Configuring the hub cluster for backup and restore
+
+You can use {ztp} to configure a set of policies to backup `BareMetalHost` resources. This allows you to recover data from a failed hub cluster and deploy a replacement cluster using {rh-rhacm-first}.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create the policy to identify `BareMetalHost` resources for backup and save it as `BareMetalHostBackupPolicy.yaml`. 
```

Comment:
change: Create the policy to identify `BareMetalHost` resources for backup and save it as `BareMetalHostBackupPolicy.yaml`.

with:
Create the policy which adds the `cluster.open-cluster-management.io/backup=cluster-activation` label to all `BareMetalHost` resources with a infraenvs.agent-install.openshift.io label.  Save this Policy under the `BareMetalHostBackupPolicy.yaml`. See below the content for the BareMetalHostBackupPolicy.yaml